src/app/service/event.service.ts

```diff
@@ -0,0 +1,25 @@
+
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { EventRequest } from '../entity/EventRequest';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EventService {
+  private addUrl = 'http://localhost:8080/api/v1/events/add';
+  private getEventsUrl = 'http://localhost:8080/api/v1/events'; // Adjust the URL as needed
+
+  constructor(private http: HttpClient) {}
+
+  addEvent(eventRequest: EventRequest): Observable<any> {
```

Comment:
Does this have to return an observable of `any`? If not, please strongly type what the observable contains